tests/test_commands.py

```diff
@@ -13,6 +13,7 @@
 FIXTURES = 'tests/fixtures'
 
 SCRIPT = 'website.py'
+MISHMASH = 'mishmash.py'
```

Comment:
Nitpick: This is easily found with a grep, not repeated a lot, not really expected to change, and easily caught by tests if set incorrectly.
Using a constant doesn't buy us much, and it adds a layer of indirection: the reader of the test needs to look up in the file to see this is a string with a filename.